### ISSUE TITLE
STIG RHEL failing on Correct file permissions with RPM

### ISSUE THREAD

**Issue by jmonson** [aws-samples/amazon-eks-custom-amis#5]
Thanks for this sample repo! I'm trying to run the RHEL STIG build, but ansible is failing.

```
    amazon-ebs: TASK [hardening : Correct file permissions with RPM] ***************************
    amazon-ebs: failed: [default] (item=audit-2.8.5-4.el7.x86_64) => {"ansible_loop_var": "item", "changed": true, "cmd": ["rpm", "--setperms", "audit-2.8.5-4.el7.x86_64"], "delta": "0:00:00.093279", "end": "2020-09-29 13:15:37.715532", "item": "audit-2.8.5-4.el7.x86_64", "msg": "non-zero return code", "rc": 1, "start": "2020-09-29 13:15:37.622253", "stderr": "chmod: cannot access ‘/var/run/auditd.state’: No such file or directory", "stderr_lines": ["chmod: cannot access ‘/var/run/auditd.state’: No such file or directory"], "stdout": "", "stdout_lines": []}
    amazon-ebs: changed: [default] => (item=cronie-anacron-1.4.11-23.el7.x86_64)
```

Steps to reproduce:
```
make AWS_REGION=us-east-1 \
    AWS_VPC_ID=vpc-XXXX \
    AWS_SUBNET_ID=subnet-XXXX \
    HARDENING=stig \
    KMS_KEY_ID=XXXX \
    build-rhel-7
```

Version:
Latest master [commit](https://github.com/aws-samples/amazon-eks-custom-amis/commit/d3e81edb1467cc75af4799f1cccba1ccdcaecf94)

Modifications:
None

Anything unusual about your environment or deployment:
I'm running this in a public subnet, using KMS.

**Comment by arhea** [comment]
@jmonson It looks like it can't find that file. Let me see why isn't there.

**Comment by arhea** [comment]
We have released version v2.0-beta which fixes this issue.
